yelp_package/Makefile

```diff
@@ -21,16 +21,26 @@ UID:=`id -u`
 GID:=`id -g`
 DOCKER_RUN_LUCID:=docker run -t -v  $(CURDIR)/../:/work:rw paastatools_lucid_container
 DOCKER_RUN_TRUSTY:=docker run -t -v  $(CURDIR)/../:/work:rw paastatools_trusty_container
-ADD_MISSING_DEPS:=-diff --unchanged-line-format= --old-line-format= --new-line-format='%L' ../requirements.txt ./extra_requirements.txt >> ../requirements.txt
+ifeq ($(PAASTA_ENV),YELP)
+	ADD_MISSING_DEPS_MAYBE:=-diff --unchanged-line-format= --old-line-format= --new-line-format='%L' ../requirements.txt ./extra_requirements_yelp.txt >> ../requirements.txt
+	ADD_YELP_PREFIX_MAYBE:=dch -v $(RELEASE)-yelp1 --changelog ../debian/changelog $$'Build for yelp: add scribereader to virtualenv'
```

Comment:
We had talked about possibly adjusting the versioning style to match Ubuntu. Do we still want to do this? That would essentially mean dropping the `-` and simply appending `yelp1` (and bumping the yelp iteration for any yelp-specific changes made on top of a particular paasta release).